### ISSUE TITLE
flocal_loss难样本挖掘策略

### ISSUE THREAD

**Issue by liushuan** [YunYang1994/tensorflow-yolov3#292]
conf_focal = tf.pow(respond_bbox - pred_conf, 2)

    conf_loss = conf_focal * (
            respond_bbox * tf.nn.sigmoid_cross_entropy_with_logits(labels=respond_bbox, logits=conv_raw_conf)
            +
            respond_bgd * tf.nn.sigmoid_cross_entropy_with_logits(labels=respond_bbox, logits=conv_raw_conf)
    )

这个 在 进行 候选框是否有物体的loss 中 引入了 focal_loss的难样本挖掘策略，不知对整体效果是否有提升
